<launch>
    <node pkg="gaja_software" type="gaja_driver.py" name="gaja_driver" required="true" output="screen">
        <param name="port_name" value="/dev/ttyUSB0"/>
        <param name="baudrate" value="9600"/>
        <param name="command_timeout" value="0.5"/>
        <param name="l_x" value="0.13"/>
        <param name="l_y" value="0.08"/>
        <param name="r" value="0.045"/>
    </node>
</launch>
